scripts/README.md

```diff
@@ -14,46 +14,83 @@ npx truffle compile
 
-For the signing process, note that Metamask underestimates gas consumption. Just make sure that the gas limit of your transaction is sufficient. There is currently no proper way to estimate the gas limit. In practice, setting the gas limit to a value higher than 5,000,000 for transactions involving 20 brackets is a practical, yet not fully reliable way to ensure a successful transaction.
+The first request will generate orders on behalf of the bracket-traders.
+Please sign this transaction in the gnosis-safe interface and double check that the prices of the orders are as intended -for example in [telegram-mainnet](https://t.me/gnosis_protocol) or [telegram-rinkeby](https://t.me/gnosis_protocol_dev) channels.
+Then continue the script and send out the second request, which will fund the bracket-traders' accounts on the exchange.
 
-### Deploy Safes
+Here is an example script invocation:
 
 ```js
-truffle exec scripts/deploy_safes.js --masterSafe=$MASTER_SAFE --fleetSize=2 --network $NETWORK_NAME
+npx truffle exec scripts/complete_liquidity_provision.js --targetToken=1 --stableToken=4 --lowestLimit=150 --hig
+hestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --investmentTargetToken=10 --investmentStableToken=1000 --fleetSize=20 --network=$NETWORK_NAME
+```
+
+This example deploys a liquidity strategy with 20 brackets between the prices 150-200 on the pair WETH-USDC.
+In this script the targetToken is 1, which happens to be WETH and the stableToken is 4, which happens to be USDC. The token ids of the exchange contract can be read from Etherscan.info in the 'Contract/Read Contract' tab, e.g. [here for mainnet](https://etherscan.io/address/0x6f400810b62df8e13fded51be75ff5393eaa841f)
+
+The fleet size should be smaller than or equal to 20, in order to ensure that the transactions can be send via Metamask - otherwise, it can happen that the payload is too high for Metamask.
+
+Please document the displayed bracket-trader addresses. They are require for the withdrawals later. They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
+
+Instead of doing all the steps with one script, the different steps can also be done individually, as explained in the next section.
+
+### Deploy safes
+
+Requires that Master is already deployed.
+
+An example of the usage would be:
+
+```js
+truffle exec scripts/deploy_safes.js --masterSafe=$MASTER_SAFE --fleet-size=20 --network=$NETWORK_NAME
 ```
 
 ### Place Orders
 
-Requires that Master and bracket Safes are already deployed.
+Requires that Master and bracket-traders are already deployed.
+
+An example of the usage would be:
 
 ```js
 truffle exec scripts/bracket_orders.js --targetToken=1 --stableToken=7 --currentPrice 270 --lowestLimit 240 --highestLimit 300 --masterSafe=$MASTER_SAFE --brackets=0xb947de73ADe9aBC6D57eb34B2CC2efd41f646636,0xfA4a18c2218945bC018BF94D093BCa66c88D3c40 --network=$NETWORK_NAME
```

Comment:
You can tell that this part of the readme is older by looking at the price of ETH. :smile: